app/helpers/sellers_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module SellersHelper
   def self.generate_seller_json(seller:)
-    locations = seller.locations
+    locations = seller.locations[0]
```

Comment:
lets rename this variable 'location'. Ideally we would change the name of the field in the JSON to 'location' as well, but that will require a simultaneous front-end change

Reply:
Dan brings up a good point. We should actually probably keep this field as-is, and add a new field called "location" that only returns the first location. Then we should migrate the frontend to use the new field, THEN delete the old "locations" field